test/integration/client-side-encryption/client_side_encryption.prose.25.lookup.test.ts

```diff
@@ -0,0 +1,413 @@
+      const collections = [
+        {
+          name: 'csfle',
+          options: { validator: { $jsonSchema: await readFixture('schema-csfle.json') } },
+          document: { csfle: 'csfle' }
+        },
+        {
+          name: 'csfle2',
+          options: { validator: { $jsonSchema: await readFixture('schema-csfle2.json') } },
+          document: { csfle2: 'csfle2' }
+        },
+        {
+          name: 'qe',
+          options: { encryptedFields: await readFixture('schema-qe.json') },
+          document: { qe: 'qe' }
+        },
+        {
+          name: 'qe2',
+          options: { encryptedFields: await readFixture('schema-qe2.json') },
+          document: { qe2: 'qe2' }
+        },
+        {
+          name: 'no_schema',
+          options: {},
+          document: { no_schema: 'no_schema' }
+        },
+        {
+          name: 'no_schema2',
+          options: {},
+          document: { no_schema2: 'no_schema2' }
+        }
+      ];
+
+      for (const { name, options } of collections) {
+        await client.db('db').createCollection(name, options);
+      }
+
+      /**
+       * Create an encrypted MongoClient configured with:
+       *
+       * ```txt
+       *   AutoEncryptionOpts(
+       *       keyVaultNamespace="db.keyvault",
+       *       kmsProviders={"local": { "key": "<base64 decoding of LOCAL_MASTERKEY>" }}
+       *   )
+       * ```
+       */
+      encryptedClient = newEncryptedClient(this);
+
+      /**
+       * ```
+       * {"csfle": "csfle"} into db.csfle
+       * Use the unencrypted client to retrieve it. Assert the csfle field is BSON binary.
+       * {"csfle2": "csfle2"} into db.csfle2
+       * Use the unencrypted client to retrieve it. Assert the csfle2 field is BSON binary.
+       * {"qe": "qe"} into db.qe
+       * Use the unencrypted client to retrieve it. Assert the qe field is BSON binary.
+       * {"qe2": "qe2"} into db.qe2
+       * Use the unencrypted client to retrieve it. Assert the qe2 field is BSON binary.
+       * {"no_schema": "no_schema"} into db.no_schema
+       * {"no_schema2": "no_schema2"} into db.no_schema2
+       * ```
+       */
+      for (const { name, document } of collections) {
+        const { insertedId } = await encryptedClient.db('db').collection(name).insertOne(document);
+
+        if (name.startsWith('no_')) continue;
+
+        expect(await client.db('db').collection(name).findOne(insertedId))
+          .to.have.property(Object.keys(document)[0])
+          .that.has.property('_bsontype', 'Binary');
+      }
+    } finally {
+      await client?.close();
+      await encryptedClient?.close();
+    }
+  });
+
+  const test = function (
+    title: string,
+    collName: string,
+    pipeline: Document[],
+    expected: Document,
+    metadata?: MongoDBMetadataUI
+  ) {
+    describe(title.slice(0, title.indexOf(':')), function () {
+      let client: MongoClient;
+
+      beforeEach(async function () {
+        Error.stackTraceLimit = 0;
```

Comment:
Leftover from debugging or is this needed for something?
```suggestion
```

Reply:
done